src/shared/codelens/javaCodeLensProvider.ts

```diff
@@ -30,12 +30,10 @@ export interface JavaLambdaHandlerComponents {
 }
 
 export async function getLambdaHandlerCandidates(document: vscode.TextDocument): Promise<LambdaHandlerCandidate[]> {
-    const manifestUri =
-        (await findParentProjectFile(document.uri, /^.*pom.xml$/)) ??
-        (await findParentProjectFile(document.uri, /^.*build.gradle$/))
-    if (!manifestUri) {
-        return []
-    }
+    const rootUri =
+        ((await findParentProjectFile(document.uri, /^.*pom.xml$/)) ??
+            (await findParentProjectFile(document.uri, /^.*build.gradle$/))) ||
```

Comment:
nit: nullish coalescing is probably better here to keep things consistent
```suggestion
            (await findParentProjectFile(document.uri, /^.*build.gradle$/))) ??
```